Source/WebCore/dom/CommandEvent.idl

```diff
@@ -26,14 +26,14 @@
 [
     EnabledBySetting=InvokerAttributesEnabled,
     Exposed=Window
-] interface InvokeEvent : Event {
-    constructor([AtomString] DOMString type, optional InvokeEventInit eventInitDict);
+] interface CommandEvent : Event {
+    constructor([AtomString] DOMString type, optional CommandEventInit eventInitDict);
 
     readonly attribute Element? invoker;
```

Comment:
I don't think we should continue calling this "invoker". But I guess that can be a follow-up? Maybe just `button`.

Reply:
Or `commandElement` as counterpart to `commandForElement`. I kinda like that as well.